packages/client/README.md

```diff
@@ -380,10 +380,10 @@ await streamr.setPermissions({
     streamId,
     assignments: [
         {
-            user: '0x12345...',
+            user: '0x1234567890123456789012345678901234567890',
             permissions: [StreamPermission.EDIT]
         }, {
-            user: '0x6789a...',
+            user: '0x6789a0x1234567890123456789012345678901234567891'
```

Comment:
Typo in the address

Reply:
It would be super readable if a end-user can think this straightforwardly:
- there are two users
- user 1 has address 0x1111... and user 2 has address 0x2222
- for user 1 we give this permission, and for user 2 we get that permission